tests/amd/amd.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @license Copyright (c) 2003-2015, CKSource - Frederico Knabben. All rights reserved.
+ * For licensing, see LICENSE.md.
+ */
+
+/* globals describe, it, expect, CKEDITOR */
+
+'use strict';
+
+CKEDITOR.define( 'testModule', [ 'ckeditor' ], function( ckeditor ) {
+	return {
+		test: ( typeof ckeditor == 'object' )
+	};
+} );
+
+describe( 'amd', function() {
+	it( 'require() should work with defined module', function( done ) {
+		CKEDITOR.require( [ 'testModule' ], function( testModule ) {
+			expect( testModule ).to.have.property( 'test' ).to.be.true();
+
```

Comment:
Unnecessary empty line (I know I'm bit*hing right now, but I think this should be consistent across our tests).